src/Core/Feature.js

```diff
@@ -185,7 +188,10 @@ class Feature {
         this.normals = options.withNormal ? [] : undefined;
         this.crs = crs;
         this.size = options.withAltitude ? 3 : 2;
-        this.extent = options.buildExtent ? defaultExtent(crs) : undefined;
+        if (options.buildExtent) {
+            this.extent = options.buildExtent ? defaultExtent(crs != 'EPSG:4978' ?  crs : options.crsIn) : undefined;
```

Comment:
You missed something here: `this.extent` will never be `undefined` in this case. 

I also don't understand the `crs` condition: could you explain a bit ?

Reply:
I change the code and add comment.